### PR TITLE
Fix building with Ogre 1.12

### PR DESCRIPTION
**Pull request by etam**
I don't know if it's compatible with earlier Ogre versions.

### CODE REVIEW

File: CMakeLists.txt [hunk 1]
```diff
@@ -92,13 +92,13 @@ if(BUILD_GAME OR BUILD_EDITOR)
         add_definitions(-DOGRE_PLUGIN_DIR_DBG="${OGRE_PLUGIN_DIR}")
     else()
         find_package(OGRE REQUIRED QUIET)
-        include_directories(${OGRE_INCLUDE_DIRS} ${OGRE_Terrain_INCLUDE_DIRS} ${OGRE_Paging_INCLUDE_DIRS})
+        include_directories(${OGRE_INCLUDE_DIRS} ${OGRE_Terrain_INCLUDE_DIRS} ${OGRE_Paging_INCLUDE_DIRS} ${OGRE_Overlay_INCLUDE_DIRS} ${OGRE_Bites_INCLUDE_DIRS})
```
Comment (paroj) [inline CMakeLists.txt:95]: these cmake changes are unnecessary, if Ogre 1.12 is properly installed, it should be taking the branch above.

**Review by paroj — CHANGES_REQUESTED** [review]

**Comment by cryham** [comment]
So @paroj is this needed if Ogre 1.13 support is merged now?

**Comment by paroj** [comment]
as far as I can see, it was not needed before either

**Comment by cryham** [comment]
Okay then, I will close this. If needed feel free to reopen.

**Pull request closed without merging**